site/content/events/2015-telaviv/_sponsors.txt

```diff
@@ -0,0 +1,74 @@
+---
+filter:    erb
+---
+<% @csponsors = [
+#	{ :image => "gigaspaces.png", :name => "GigaSpaces Technologies", :link => "http://www.gigaspaces.com"},
+]%>
+
+<% unless @csponsors.empty?  %><h1>Event Leaders</h1><% end %>
+
+<% @csponsors.each do |sponsor| %>
+<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=100px height=100px src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
+<% end %>
+
+<br/>
+<br/>
+
+<% @platinum_sponsors = [
+#	{ :image => "aolon.png", :name => "AOL On", :link => "http://on.aol.com/"},	
```

Comment:
it's a comment, just a place holder to show how to fill this field.

On 5/17/15, Uri Cohen <notifications@github.com> wrote:
>> +---
>> +<% @csponsors = [
>> +#	{ :image => "gigaspaces.png", :name => "GigaSpaces Technologies", :link
>> => "http://www.gigaspaces.com"},
>> +]%>
>> +
>> +<% unless @csponsors.empty?  %><h1>Event Leaders</h1><% end %>
>> +
>> +<% @csponsors.each do |sponsor| %>
>> +<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>'
>> title='<%= sponsor[:name] %>' width=100px height=100px src='<%=
>> File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
>> +<% end %>
>> +
>> +<br/>
>> +<br/>
>> +
>> +<% @platinum_sponsors = [
>> +#	{ :image => "aolon.png", :name => "AOL On", :link =>
>> "http://on.aol.com/"},	
>
> Is AOL sponsoring this year or is it a leftover?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/jedi4ever/devopsdays-webby/pull/1733/files#r30472357